pennylane/math/single_dispatch.py

```diff
@@ -456,6 +456,15 @@ def _cond_tf(pred, true_fn, false_fn, args):
 )
 
 
+def _pad_tf(tensor, paddings, mode="CONSTANT", constant_values=0, name=None):
+    if ar.ndim(paddings) == 1:
+        paddings = (paddings,)
+    return _i("tf").pad(tensor, paddings, mode=mode, constant_values=constant_values, name=name)

```

Comment:
I think it creates an error for one test:

```
FAILED tests/pauli/test_conversion.py::TestPhasedDecomposition::test_differentiability[matrices0-False] - TypeError: pad(): argument 'pad' (position 2) must be tuple of ints, but found element of type tuple at pos 0
```

Reply:
fixed!